app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_age() -> None:
+    assert get_human_age(0, 0) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(0, 0)}"
+    assert get_human_age(0, 10) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(0, 10)}"
+    assert get_human_age(10, 0) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(10, 0)}"
```

Comment:
The test `test_zero_age` seems to be testing the same case multiple times with different inputs but expecting the same output. According to the checklist, 'Don't Repeat Yourself', ensure that your tests don't test the same case multiple times unless necessary.